tendrl/monitoring_integration/manager/__init__.py

```diff
@@ -0,0 +1,128 @@
+import __builtin__
+import traceback
+import sys
+import json
+
+import signal
+import maps
+import gevent
+import os
+
+from tendrl.monitoring_integration.grafana import utils
+from tendrl.monitoring_integration.grafana import exceptions
+from tendrl.monitoring_integration.grafana import dashboard
+from tendrl.monitoring_integration.grafana import datasource
+from tendrl.commons import manager as common_manager
+from tendrl import monitoring_integration
+from tendrl.monitoring_integration import sync
+from tendrl.commons import TendrlNS
+
+
+class MonitoringIntegrationManager(common_manager.Manager):
+
+    def __init__(self):
+
+        self._complete = gevent.event.Event()
+        super(
+            MonitoringIntegrationManager,
+            self
+        ).__init__(
+            sync.MonitoringIntegrationSdsSyncThread()
+        )
+
+    def start(self):
+        pass
+
+    def stop(self):
+        pass
+
+
+def upload_default_dashboards():
+
+        monitoring_integration_manager = MonitoringIntegrationManager()
+
+        # Creating Default Dashboards
+        dashboards = []
+        config = utils.get_conf("/etc/tendrl/monitoring-integration/" +
+                                "monitoring-integration.conf.yaml")
+        setattr(NS, "conf", config)
+
+        dashboards = dashboard.get_all_dashboards()
+
+        title = []
+
+        for dashboard_json in dashboards:
+            title.append(dashboard_json["uri"].split('/')[1])
+
+        for dashboard_json in config.dashboards:
+            if dashboard_json in title:
+                sys.stdout.write('\n' + "Dashboard " + str(dashboard_json) +
```

Comment:
Youll need to use https://github.com/Tendrl/commons/blob/master/tendrl/commons/utils/log_utils.py#L10

Instead of sys.stdout.write everywhere